README.md

```diff
@@ -8,8 +8,8 @@
 [![](http://transparent-favicon.info/favicon.ico)](#)
 [community.chocolatey.org profile](https://community.chocolatey.org/profiles/chocolatey-community)
 
-# NOTE: Currently we are unlikely to accept any new or migrated packages into this repository. Please raise a [Discussion](discussions) if you have a strong case for new or migrated packages.
-# NOTE: The repository is currently being maintained by community members of the Chocolatey Team in their spare time. If you are interested in helping out [troubleshoot issues](issues), review [pull requests](pulls) or contribute code for the maintenance of packages, we would be happy to have you! If you have any questions, please raise a [discussion](discussions) or reach out on our [Community Chat](https://ch0.co/community).
+# NOTE: Currently we are unlikely to accept any new or migrated packages into this repository. Please raise a [Discussion](../../discussions) if you have a strong case for new or migrated packages.
+# NOTE: The repository is currently being maintained by community members of the Chocolatey Team in their spare time. If you are interested in helping out [troubleshoot issues](../../issues), review [pull requests](../../pulls) or contribute code for the maintenance of packages, we would be happy to have you! If you have any questions, please raise a [discussion](../../discussions) or reach out on our [Community Chat](https://ch0.co/community).
```

Comment:
I appreciate the time you have taken to fix up these links.

With that said, I think we should try to avoid relative links in the Readme. Can you please update them to be full absolute links instead?

Reply:
OK, I've updated the links